From: Dan Callaghan <djc@djc.id.au>
Upstream: TODO

use POSIX strncasecmp

Not sure what this is about, strncasecmp is not a member of std::char_traits in 
any STL I can find...

--- gmrun-0.9.2-orig/src/ci_string.h	2001-05-17 00:39:31.000000000 +1000
+++ gmrun-0.9.2/src/ci_string.h	2010-10-09 15:39:27.734709225 +1000
@@ -26,7 +27,7 @@
   static int compare( const char* s1,
                       const char* s2,
                       size_t n ) {
-    return ::strncasecmp( s1, s2, n );
+    return strncasecmp( s1, s2, n );
   }
 
   static const char*
